src/main/java/lol/fairplay/ghidraapple/analysis/utilities/programUtils.kt

```diff
@@ -45,6 +45,7 @@ fun dataBlocksForNamespace(program: Program, ns: Namespace, addresses: AddressSe
     return dataBlocks
 }
 
+@Deprecated("Use parseObjCListSection instead.")
```

Comment:
I think that we are still in an early phase of development where we can just remove deprecated code directly